Updated to ChatFormat

cyberkeiler opened the pull request:
Colors and TextFormat deprecated - updated to ChatFormat:
LIGHT_GREEN, GREEN and Orange weren't available, so I translated LIGHT_GREEN to GREEN, GREEN to DARK_GREEN and ORANGE to GOLD ;)

svdragster committed:
Create README.md

svdragster committed:
Create SafeTrade.jar

svdragster committed:
Create SafeTradeListener.java

svdragster committed:
Rename SafeTrade.jar to SafeTrade.java

cyberkeiler committed:
Updated to ChatFormat

Had to translate LIGHT_GREEN to GREEN and GREEN to DARK_GREEN

cyberkeiler committed:
added Canary.inf

svdragster commented:
Thanks, however the whole plugin isn't working since 1.8. I will get back to it soon

cyberkeiler commented:
actually it is ;) Running on CanaryMod-1.2.0.
Great plugin by the way! - I just corrected the package structure and added the Canary.inf - nothing else to change!

svdragster commented:
Oh really? Someone tested it and told me it's not working correctly, but that may be because of a canary issue which got fixed. Thanks for the feedback!
